foods.html

```diff
@@ -5,7 +5,22 @@
   <title>Quantified Self - Manage Foods</title>
 </head>
 <body>
+  <table id='food-table'>
+    <tr>
+      <th>Name</th>
+      <th>Calories</th>
+    </tr>
+  </table>
 
+  <form>
+    Food Name:<br>
+    <input type="text" name="new-food-name" id="new-food-name"><br>
+    Calorie Count:<br>
+    <input type="text" name="new-food-calories" id="new-food-calorie-count"><br>
```

Comment:
Let's add a validation check to make sure that both of these fields are filled so that incomplete food data isn't getting posted!